game-core/src/test/java/games/strategy/triplea/ui/panels/map/MapRouteDrawerTest.java

```diff
@@ -101,4 +107,17 @@ void testCorrectParameterHandling() {
     verify(dummyRouteDescription, times(2)).getRoute();
     verify(dummyRouteDescription.getRoute(), atLeastOnce()).getAllTerritories();
   }
+
+  /** Regression test for https://github.com/triplea-game/triplea/issues/7112 */
```

Comment:
References are good, 'required references' are not. What would happen if every test case required you to go to a different issue to understand why the test case was there? I think duplicating some of the details from the issue will be enough so that you don't have to look at the issue to understand this (though the reference being there is good nonetheless so if someone wants even more details, they can get them).